README.md

```diff
@@ -7,6 +7,13 @@
 # Introduction
 Official repository for Semantic Web for Earth and Environmental Terminology (SWEET) Ontologies.
 
+[View in pyLODE](https://sweetontology.net/sweetAll) &nbsp; 
```

Comment:
You are trying to connect to an HTTPS URL... we do not have a cert for sweetontology.net. This needs to be changed to `http://sweetontology.net/sweetAll`

Reply:
fixed, sorry, thanks